applications/teletype/src/templates/teletype_new_account.erl

```diff
@@ -71,6 +71,8 @@ handle_new_account(JObj, _Props) ->
 -spec process_req(wh_json:object()) -> 'ok'.
 -spec process_req(wh_json:object(), wh_proplist()) -> 'ok'.
 process_req(DataJObj) ->
+    %% Give onboarding tools time to add initial users...
+    timer:sleep(30000),
```

Comment:
I would put this on the `cb_accounts` side personally; maybe `timer:apply_after/4` on the [publish](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/modules/cb_accounts.erl#L1465) (which should probably use a call to wh_amqp_worker as well).

My 2 cents